packages/fxa-content-server/server/bin/fxa-content-server.js

```diff
@@ -59,8 +59,10 @@ const fourOhFour = require('../lib/404');
 const serverErrorHandler = require('../lib/500');
 const localizedRender = require('../lib/localized-render');
 const csp = require('../lib/csp');
-const cspRulesBlocking = require('../lib/csp/blocking')(config);
-const cspRulesReportOnly = require('../lib/csp/report-only')(config);
+const cspRulesBlocking = require('../../../fxa-shared/csp/blocking')(config);
```

Comment:
I don't think you can easily move things to another package like `fxa-shared`, currently. [You'll probably run into being unable to use `../` to refer to files outside the current package directory when we run things in Docker](https://github.com/mozilla/fxa/issues/941#issuecomment-505511291). Might run locally, but will hit snags in testing and building for deployment.

Reply:
Sorry, I should have mentioned Phil's PR: https://github.com/mozilla/fxa/pull/1860

I should be able to do this once that is merged, right?

Reply:
Sorry @LZoog, looks like I steered you in the direction of #1860 without understanding the limitations of that approach. @philbooth, which approach to accessing fxa-shared would be best for this PR?

Reply:
Huh well, I read through it and thought myself that it would be possible to do this with that landed! I commented out a section in `fxa-shared/index.js` that sets it up the current/old way - I can go that route if this won't work.

Reply:
Looks like #1860 won't solve this problem generally. That PR partly solves it for fxa-content-server, by jumping up a directory and [explicitly copying fxa-shared into the container with fxa-content-server](https://github.com/mozilla/fxa/pull/1860/files#diff-bdd3d89f8c2822d921c755a374232952R30). You'd have to replicate those tweaks here for it to work for fxa-payment-server

Reply:
Ah, I see. That makes sense.
Since we'd said on Slack that option #3 is the best way to go anyway, I'll remove the shared stuff and create the separate middleware. I won't need to reference the shared server at all.